js/ad_manager_vast.js

```diff
@@ -697,17 +851,27 @@ OO.Ads.manager(function(_, $) {
           indexInPod: adWrapper.ad.adPodIndex,
           skippable: false
         });
-      }
-      else {
+      } else {
         var streamUrl;
         if (adWrapper.ad && adWrapper.ad.streamUrl) {
           streamUrl = adWrapper.ad.streamUrl;
         }
         else if (adWrapper.streamUrl) {
           streamUrl = adWrapper.streamUrl;
+        } else {
+          streamUrl = '';
         }
+
         this.amc.sendURLToLoadAndPlayNonLinearAd(adWrapper, adWrapper.id, streamUrl);
-        this.checkCompanionAds(adWrapper.ad);
+      }
+      this.checkCompanionAds(adWrapper.ad);
+      if (isVPaid) {
+        //Since IMA handles its own UI, we want the video player to hide its UI elements
```

Comment:
Is this comment copy/pasted from the IMA code? I don't see a reason to reference IMA here.